components/tag/stories/template.js

```diff
@@ -55,15 +55,15 @@ export const Template = ({
 						size: "50",
 				  })
 				: ""}
-			${iconName
+			${iconName && !avatarUrl
```

Comment:
If I try to add an icon in Storybook on the Avatar variant, both the avatar and the icon disappear. The Avatar isn't an available control in Storybook so we should probably remove the icon from showing up as a control for the Avatar variation if that's not allowed to be added

Reply:
Per @mlogsdon18's feedback, you can add `if: { arg: 'iconName', truthy: false },` to avatarUrl and `if: { arg: 'avatarUrl', truthy: false },` to iconName to make them mutually exclusive